packages/suite-desktop-core/e2e/support/fixtures.ts

```diff
@@ -37,24 +38,42 @@ const test = base.extend<Fixtures>({
     trezorUserEnvLink: async ({}, use) => {
         await use(TrezorUserEnvLink);
     },
-    electronApp: async ({ trezorUserEnvLink, startEmulator, emulatorConf }, use) => {
+    appContext: async (
+        { trezorUserEnvLink, startEmulator, emulatorConf, locale, colorScheme },
+        use,
+        testInfo,
+    ) => {
         // We need to ensure emulator is running before launching the suite
         if (startEmulator) {
             await trezorUserEnvLink.stopBridge();
+            await trezorUserEnvLink.stopEmu();
             await trezorUserEnvLink.connect();
             await trezorUserEnvLink.startEmu({ wipe: true });
             await trezorUserEnvLink.setupEmu(emulatorConf);
         }
-        const suite = await launchSuite();
-        await use(suite.electronApp);
-        await suite.electronApp.close(); // Ensure cleanup after tests
+
+        if (testInfo.project.name === PlaywrightProjects.Desktop) {
+            const suite = await launchSuite({ locale, colorScheme });
+            await use(suite.electronApp);
+            await suite.electronApp.close(); // Ensure cleanup after tests
+        } else {
+            if (startEmulator) {
+                await trezorUserEnvLink.startBridge();
+            }
+            await use(undefined);
+        }
     },
-    window: async ({ electronApp }, use, testInfo) => {
-        const window = await electronApp.firstWindow();
-        await window.context().tracing.start({ screenshots: true, snapshots: true });
-        await use(window);
-        const tracePath = `${testInfo.outputDir}/trace.electron.zip`;
-        await window.context().tracing.stop({ path: tracePath });
+    window: async ({ appContext, page }, use, testInfo) => {
+        if (appContext) {
+            const window = await appContext.firstWindow();
+            await window.context().tracing.start({ screenshots: true, snapshots: true });
+            await use(window);
+            const tracePath = `${testInfo.outputDir}/trace.electron.zip`;
+            await window.context().tracing.stop({ path: tracePath });
+        } else {
+            await page.goto('./');
+            await use(page);
+        }
```

Comment:
combination of web/electron and starting or not the emulator clatters these fixtures a bit. I imagine as we convert more and more tests, we will have better understanding what we need and how. Then we should refactor this crucial part to be a cleaner and simpler